CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 0.18.0 2025-02-13
```

Comment:
Remember to also update the version in api.json too (otherwise sdk won't trigger a rebuild)